js/ui/components/networkAgent.js

```diff
@@ -219,7 +226,7 @@ var NetworkSecretDialog = new Lang.Class({
         if (userSetting) {
             let allowDownloads = userSetting.get_data(NM_SETTING_ALLOW_DOWNLOADS);
             if (allowDownloads)
-                defaultValue = (allowDownloads === '0');
+                defaultValue = (allowDownloads === '1');
```

Comment:
This PR also changes what's supposed to be the default from "not allowing downloads" to "allowing downloads", which I don't see documented neither in the commit message nor in the originating ticket. Where is this coming from?

If it's not  a mistake, please add a brief comment to the commit message.